src/main/java/com/laserfiche/repository/api/clients/AttributesClient.java

```diff
@@ -1,9 +1,16 @@
 package com.laserfiche.repository.api.clients;
 
-import com.laserfiche.repository.api.clients.impl.model.Attribute;
-import com.laserfiche.repository.api.clients.impl.model.ODataValueContextOfListOfAttribute;
+import java.util.*;
+import java.util.function.Function;
+import java.util.stream.Collectors;
```

Comment:
We probably don't need these imports for interfaces. I forgot to mention that you could run "optimize imports" in batch to remove unused imports.

Reply:
I removed unnecessary imports form clients (interfaces and their implementations).